Library/Homebrew/extend/ARGV.rb

```diff
@@ -97,6 +97,7 @@ def force?
     flag? "--force"
   end
 
+
```

Comment:
😦I don't have this locally and `git status` doesn't show there are unstaged or uncommited changes for `ARGV`. Probably it'll disappear when I fix all the others comments and rebase.